iphone/Classes/TiUITableView.h

```diff
@@ -84,6 +83,8 @@
 }
 
 @property (nonatomic, assign) BOOL viewWillDetach;
+@property (nonatomic, assign) BOOL isLoaded;
```

Comment:
Can you be more specific here with the variable name? What exactly is `isLoaded` associated with?